docusaurus/express/API/Security/CORS.mdx

```diff
@@ -8,30 +8,9 @@ sidebar_position: 3
 Cross-origin resource sharing (CORS) is a mechanism that allows restricted resources on a web page to be requested from another domain outside the domain from which the first resource was served.
```

Comment:
I'd mention that this policy is enforced by web browsers, but configured by the backend server.